### ISSUE TITLE
Basic Examples Open by Default

### ISSUE THREAD

**Issue by smnpaorg** [jmlopez-rod/ApplicationMaker#2]
Most of the "Basic Examples" in the documentation center are open by default.

How do I have my custom documentation display the same Basic Examples open by default?

**Comment by jmlopez-rod** [comment]
I can't remember if this is done by simply saving the file with the examples already opened. Have you tried saving it like that?

**Comment by smnpaorg** [comment]
@jmlopez-rod Yes you can't do such because the arrows don't appear by default if you open the ___ editable versions of the files.

**Comment by jmlopez-rod** [comment]
@smnpaorg If the arrows don't appear maybe it is missing the option `ShowGroupOpener->True`. Check the cell to see if it has this option. Then try saving it when the group is open. Once you export I believe the look of the notebook should remain the same.

**Comment by smnpaorg** [comment]
@jmlopez-rod Unfortunately `ShowGroupOpener->True` doesn't seem to make a difference.
